src/main/scala/QuartzSchedulerExtension.scala

```diff
@@ -383,7 +383,7 @@ class QuartzSchedulerExtension(system: ExtendedActorSystem) extends Extension {
    */
   protected def scheduleJob(name: String, receiver: AnyRef, msg: AnyRef, startDate: Option[Date])(schedule: QuartzSchedule): Date = {
     import scala.collection.JavaConverters._
-    log.info("Setting up scheduled job '{}', with '{}'", name, schedule)
+    log.debug("Setting up scheduled job '{}', with '{}'", name, schedule)
```

Comment:
What's the reason for changing this from `info` to `debug`?

Reply:
Why would you like to keep it ```info``` ? (Please see other existing debug statements in this class)

Reply:
Oh, I should have phrased that differently. I was merely asking because I was curious for the reason for the change. I see that some of the other statements are debug. Let's just leave this one as `info` for now.